### PR TITLE
[#9773] Student course detail page: Standardize picture size and title length

### LINKED ISSUE #9773: Student course detail page: Standardize picture size and title length
- **Environment**:  `master` branch at commit ad2d071317e2b1c70139daf43148219cf859f0db

**Steps to reproduce**
1. login as a student who has teammates, some teammates have their profile photo whose size is large
2. click the `View team` button in home page

**Actual behaviour**
Captureccccccc

**Expected behaviour**
the picture and title length should be standardized so that the details of students are aligned.

### PR DESCRIPTION
**Pull request by crizyli**
Fixes #9773 

Ready for review.

### CODE REVIEW

File: src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss [hunk 1]
```diff
@@ -7,7 +7,12 @@
 }
 
 .profile-pic {
-  height: 35%;
+  height: 130px;
```
Comment (xpdavid) [inline src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss]: The table will not be responsive by doing this?

File: src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss [hunk 2]
```diff
@@ -7,7 +7,12 @@
 }
 
 .profile-pic {
-  height: 35%;
+  height: 50%;
+  width: 85%;
+}
+
+.title-fixed-length {
+  width: 305px;
```
Comment (xpdavid) [inline src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss]: Same here?

**Comment by xpdavid** [comment]
@crizyli Can you post a screenshot showing the changes in UI?

**Comment by crizyli** [comment]
> @crizyli Can you post a screenshot showing the changes in UI?

image

File: src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss [hunk 3]
```diff
@@ -7,7 +7,12 @@
 }
 
 .profile-pic {
-  height: 35%;
+  width: 85%;
+  height: 50%;
```
Comment (erjillsison) [inline src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss]: Would it be better to use `max-width: ?` and `max-height: ?` to maintain aspect ratio of image?

File: src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss [hunk 4]
```diff
@@ -7,7 +7,12 @@
 }
 
 .profile-pic {
-  height: 35%;
+  max-width: 85%;
+  max-height: 50%;
+}
+
+.title-fixed-length {
```
Comment (xpdavid) [inline src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss]: A percentage is not technically "fixed"

**Comment by wkurniawan07** [comment]
Will be closing this as the linked issue has been resolved.

**Pull request closed without merging**